### PR TITLE
strona_Authors

### PR DESCRIPTION
**Pull request by mikolaj-sar**
Hej, Strona "authors" zrobiona w odrębnym pliku html. Do pliku main.css wkleiłem elementy kodu do strony.

### CODE REVIEW

File: authors.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        </div>
+      </div> 
+      <div class="information-authors">
+        <div class="personal-data">
+            <img class="Karol-image" src="Ikony/zielony proporczyk.png">
+            <h2 class="carousel-paragraph container-fluid">Karol Nogal</h2>
+            <p class="personal-data-text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Lorem ipsum dolor sit amet consectetur adipisicing elit. Lorem ipsum dolor sit amet consectetur adipisicing elit. Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
+        </div>
+        <div class="personal-data">
+            <img class="Maxymilian-image" src="Ikony/niebieski proporczyk.png">
```
Comment (krichert) [inline authors.html]: use english, please do not make spaces in file names

File: main.css [hunk 2]
```diff
@@ -98,4 +98,47 @@ body{
...
\ No newline at end of file
+}
+
+/* AUTHORS - CSS code*/
+
+.section-authors {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    height: 87%;
```
Comment (krichert) [inline main.css]: why 87%? it is not common value

File: main.css [hunk 3]
```diff
@@ -98,4 +98,47 @@ body{
...
+    width: 100%;
+    background-color: brown;
+    text-align: center;
+}
+
+.section-authors-title {
+    font-family: ‘LABRIT’;
```
Comment (krichert) [inline main.css]: use font-family if the  first one will not load ->  e.g. `font-family: ‘LABRIT’, sans-serif;`

**Review by krichert — CHANGES_REQUESTED** [review]

**Comment by Tarnoruda** [comment]
This is older version, rejecting.

**Pull request closed without merging**